README.md

```diff
@@ -16,10 +16,24 @@ Android Engineer Interview Questions
 ## Android
  - How does Garbage collection works?
  - What is a dangling pointer?
+ - Explain the terms reference counting, mark and sweep? 
```

Comment:
I think we can skip this question as this is something very rare in interviews. The idea of this repository is to collect best possible and most occurring questions.

Reply:
I was never asked about dangling pointer ever. Perhaps we can remove that.